.github/workflows/go.yml

```
@@ -307,9 +306,10 @@ jobs:
     if: github.ref != 'refs/heads/main'
     steps:
       - uses: actions/checkout@v2
-
       - name: codegen
-        run: make pkg/assets/zz_generated_offsets_linux.go
+        run: |
+          make pkg/assets/zz_generated_offsets_linux.go
+          make static/gen_manifests.go
```

```suggestion
          make -j$(nproc) pkg/assets/zz_generated_offsets_linux.go static/gen_manifests.go
```

They can be generated in parallel

---

Makes sense, I have made these changes. 
I could only test the CI pipeline by committing to the repo, hence the sea of commits :)
I've squashed them now

---

yeah, its perfectly fine while getting feedback and working on the PR. But its nice that the git history of the final result is clean and tidy. Thanks!